project/Dependencies.scala

```diff
@@ -116,6 +118,13 @@ object ExternalDependencies {
   val awsLambdaClient: Seq[ModuleID] = Seq(
     "com.amazonaws" % "aws-lambda-java-runtime-interface-client" % versions.awsLambdaClient
   )
+  val awsLambdaEvents = Seq(
+    "com.amazonaws" % "aws-lambda-java-events" % versions.awsLambdaEvents
+  )
+
+  val uPickle = Seq(
+    "com.lihaoyi" %% "upickle" % versions.uPickle
```

Comment:
Are we using this lib?

Reply:
It provides ujson.  It's the same library we use in the concepts API for reading JSON.

Reply:
Ah, what got me is that I was looking for imports from `com.lihaoyi`, but `ujson` is accessible globally (or probably more correctly the library is namespaced `ujson`?)